app/facades/cocktail_facade.rb

```diff
@@ -52,7 +53,7 @@ def self.retrieve_search_results(query, user_id)
     response = CocktailService.search_cocktails(query)
 
     if response[:drinks].nil?
-      return false
+      false
     else
       response[:drinks].map do |drink|
         {
```

Comment:
I'd like to look deeper into this. The first line of the method is needed (and needed in that location) for the details end point and handles nil values here so something is off about this code. 🤔

Reply:
Ahh it's in the search method here. I suggest refactoring this to use a guard clause, since I think it'd be a lot easier to read, but that's def a nit pick. Seems functional.

Reply:
BUT also I made a self.find_rating method you can use for the end of this method. Suggested refactor as well! (Non-critical)